lib/proofer/applicant.rb

```diff
@@ -1,5 +1,6 @@
 module Proofer
   class Applicant
+    attr_accessor :uuid
```

Comment:
why are these on separate lines? I'd expect to see a list of attributes rather than a separate `attr_accessor` on each line